### PR TITLE
[Chore] Cleanup with more lint rules

### PR DESCRIPTION
**Pull request by RogerKSI**
## Implementation details
- add more lint rules
- fix lint based on new rules

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [ ] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [ ] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: x/bank/keeper/keeper.go [hunk 1]
```diff
@@ -71,7 +70,7 @@ func (k WrappedBankKeeper) BurnCoins(ctx sdk.Context, moduleName string, amt sdk
 	}
 
 	// Instead of burning coins, we send them to the community pool.
-	k.SendCoinsFromModuleToModule(ctx, moduleName, distrtypes.ModuleName, amt)
+	_ = k.SendCoinsFromModuleToModule(ctx, moduleName, distrtypes.ModuleName, amt)
```
Comment (taobun) [inline x/bank/keeper/keeper.go:73]: Should return error?

File: yoda/executor/rest_test.go [hunk 2]
```diff
@@ -86,7 +86,7 @@ func TestExecuteResponseNotOk(t *testing.T) {
 }
 
 func TestExecuteFail(t *testing.T) {
-	testServer := creatExecuteFailSenarioServer()
+	testServer := createExecuteFailSenarioServer()
```
Comment (taobun) [inline yoda/executor/rest_test.go:89]: Scenario

File: x/bank/keeper/keeper.go (hunk 1 above)
Comment (RogerKSI) [reply x/bank/keeper/keeper.go]: updated

File: yoda/executor/rest_test.go (hunk 2 above)
Comment (RogerKSI) [reply yoda/executor/rest_test.go]: updated

**Pull request merged**